src/permissions.ts

```diff
@@ -0,0 +1,4 @@
+export * as PermissionTypes from './lib/permissions/interfaces';
```

Comment:
Idea: Export anything permission-related from core/permissions

How do  these grouped types look? I think it might lead to excessive verbosity